src/schemas/json/partial-poetry.json

```diff
@@ -548,16 +548,8 @@
     "dependencies": {
       "type": "object",
       "description": "This is a hash of package name (keys) and version constraints (values) that are required to run this package.",
-      "required": ["python"],
-      "properties": {
-        "python": {
-          "$ref": "#/definitions/poetry-dependency",
-          "type": "string",
-          "description": "The Python versions the package is compatible with."
-        }
-      },
       "patternProperties": {
-        "^(?!python$)[a-zA-Z-_.0-9]+$": {
```

Comment:
Do you think this change should be a separate PR? I wouldn't be surprised if this fixes validation errors when a person wants to specify a non-cpython version of Python.

Reply:
Yes, it is required.

With python no longer required, it is simpler to write.

And another problem is that some libraries do not support lookahead regular expressions, so this fix is more desirable.

Create another PR.

Reply:
I see, thank you for the info

Reply:
https://github.com/SchemaStore/schemastore/pull/4450